### DISCUSSION TITLE
PKGi PSP v1.0.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/pkgi-psp#9] [Announcements]
### Added

* Package install as `ISO`/`CSO`/`Digital`
* PSP Go internal storage detection (`ef0`/`ms0`)
* Update database files from URLs (`Refresh` option)
* Enabled `Search` option (on-screen keyboard)
* Added PSX category
* PKGi new version check & notification

### Fixed

* Install and decrypt themes to `/PSP/THEME/`

### Misc

* Improved download speed (up to 500 Kb/s)
* Network proxy settings support

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/pkgi-psp/releases/tag/v1.0.0'>PKGi PSP v1.0.0</a>.</em>

**Comment by DJTOMATO** [comment]
just says corrupted data 6.61 pro-c inf

**Comment by Camilooo1637** [comment]
Cómo los instalo
